src/models/wallet.ts

```diff
@@ -55,6 +56,32 @@ export class Wallet{
         return aliasName ? aliasName : Helper.createAddress(address).pretty();
     }
 
+    convertAddressToNamePretty(address: string, includeOthers: boolean = false): string{
```

Comment:
Hi Chun Lam, if you look at my previous PR, I have updated the address to Pretty. Dont think we need one without returning with pretty is it? or can we merge them into one function by giving it an option?

Reply:
actually, should have option to have both plain or pretty. All our address property in our model is storing only plain, you will have a hard time comparing later

Reply:
we need both